php/attributes/default.rb

```diff
@@ -59,3 +59,11 @@
 default['amazon-elasticache-cluster-client'] = {
   'php_version' => '7.0'
 }
+
+default['poppler'] = {}
+default['poppler']['package_uri'] = 'https://packagecloud.io/till/poppler-backport/ubuntu/'
+default['poppler']['package_arch'] = 'amd64'
+default['poppler']['package_distro'] = 'trusty'
+default['poppler']['package_components'] = ['main']
+default['poppler']['package_key_uri'] = 'https://packagecloud.io/till/poppler-backport/gpgkey'
```

Comment:
Can you import the key into cookbook files? See how I did it with amazon-elasticache-client.